install.sh

```diff
@@ -367,6 +378,7 @@ else
         m)    mininet;;
         n)    ndn;;
         p)    patchDummy;;
+        q)    quiet_install;;
```

Comment:
Order for quiet_install is important, right? i.e. if we specify after -a then it would be a problem.
Is there some way to make it execute first. At least we can mention in the help that it needs to go first.

I/other members of Mini-NDN don't have access to merge patches from GitHub. If you could push to gerrit.named-data.net/ that would be great, see https://github.com/named-data/NFD/blob/master/CONTRIBUTING.md#gerrit-uploading-patches-and-code-review. Else we can finish the review here and I can push this change to gerrit (preserving your authorship).